wordpress-mobile/WordPress-Android
# [Reader IA] Subscribing to tag doesn't update the chip count

Fixes #20012 

-----

## To Test

1. Install Jetpack and sign in;
2. Open Reader;
3. Open the drop-down menu and tap on `Subscriptions` feed;
4. Tap the tags pill;
5. Subscribe to a tag;
6. Go back to reach the Subscriptions feed again
7. Verify the number of tags is shown correctly
8. Repeat steps 4-7 but this time unsubscribe from a tag.



-----

## Regression Notes

1. Potential unintended areas of impact

    - None

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing

3. What automated tests I added (or what prevented me from doing so)

    --
-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
